orchagent/pfcactionhandler.h

```diff
@@ -58,6 +79,7 @@ class PfcWdActionHandler
         uint8_t m_queueId = 0;
         shared_ptr<Table> m_countersTable = nullptr;
         PfcWdQueueStats m_stats;
```

Comment:
>m_stats [](start = 24, length = 7)

call it m_queuestats?

Reply:
No need to keep it as a class member anymore. Thanks for the suggestion.